box/config.yml

```diff
@@ -1287,7 +1291,7 @@ php_packages_extra:
 # XDebug configuration.
 php_xdebug_default_enable: 1
 php_xdebug_cli_disable: no
-php_xdebug_remote_autostart: 1
+php_xdebug_remote_autostart: 0
```

Comment:
Was this meant to be in here?

Reply:
DANG IT! The local.blt.yml file didn't fix it updating this. THANK YOU, CORY! Agh